admin-dev/themes/new-theme/js/components/multistore-form.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+import ComponentsMap from '@components/components-map';
+import {ConfirmModal} from '@components/modal/confirm-modal';
+
+const {$} = window;
+
+const initMultistoreForm = () => {
+  const MultistoreFormMap = ComponentsMap.multistoreForm;
+  const MultistoreHeaderMap = ComponentsMap.multistoreHeader;
+  const multistoreForm = $(MultistoreFormMap.formRow).closest('form');
+  const $modalItem = $(MultistoreHeaderMap.modal);
+
+  const generateFormValuesHash = () => {
+    const formValues = multistoreForm.serializeArray().reduce((obj: any, item: any) => {
+      const form = multistoreForm[0] as HTMLFormElement;
+      const formElement = form.elements[item.name] as HTMLFormElement;
+
+      if ($(formElement).prop('disabled')) { // Ignore if element is disabled
+        return obj;
+      }
+      const fieldValues = obj;
+
+      if (formElement instanceof HTMLTextAreaElement) {
+        fieldValues[item.name] = item.value
+          .replace(/<[^>]*>?/gm, '') // remove html tags
+          .replace(/\r?\n|\r/g, '') //remove line breaks
+          .replace(/(\.\s)/g, '.'); //remove space after dots added by TinyMCE
```

Comment:
```suggestion
          .replace(/<[^>]*>?/gm, '') // remove html tags
          .replace(/\r?\n|\r/g, '') //remove line breaks
          .replace(/(\.\s)/g, '.'); //remove space after dots added by TinyMCE
```

If you do this, it means that if I modify some HTML tags inside tinyMCE, I can quit the page without having the confirmation modal, which is wrong

Reply:
I also have doubts regarding his cleaning of TinyMCE's content 🤔. Comparing the text content only can make sense in other use cases but here the HTML content should also be considered

Reply:
On page load, the values in textarea is different of the TinyMCE content. The value in the textarea is the raw value, Tinymce enrich the original value by adding HTML tags and formatting like a space after the dots

That's why I clean them all before comparing values

Reply:
But when you save then all the additional content added by TinyMCE is indeed saved in the DB no?
So it's only the initial data from the fixtures which don't match and after a save I believe it is the same?

Besides with this modif I still don't see how you can see any difference when only the HTML tags are modified 🤔